megamek/src/megamek/common/Infantry.java

```diff
@@ -114,26 +90,19 @@ public class Infantry extends Entity {
     private boolean sneak_ir = false;
     private boolean sneak_ecm = false;
 
-    /**
-     * Stores which infantry specializations are active.
-     */
+    /** The active specializations of this platoon. */
     private int infSpecs = 0;
 
     /**
      * For mechanized VTOL infantry, stores whether the platoon are microlite troops,
      * which need to enter a hex every turn to remain in flight.
      */
-    private boolean microlite = false;
+    private boolean isMicrolite = false;
 
-    /**
-     * The location for infantry equipment.
-     */
     public static final int LOC_INFANTRY = 0;
     public static final int LOC_FIELD_GUNS = 1;
 
-    /**
-     * Infantry only have critical slots for field gun ammo
-     */
+    // Infantry only have critical slots for field gun ammo
     private static final int[] NUM_OF_SLOTS = { 20, 20 };
     private static final String[] LOCATION_ABBRS = { "MEN", "FGUN" };
```

Comment:
Since we're polishing this, can we change the "MEN" location to "troopers" or "soldiers" or something?

Reply:
I like troopers myself. But agree with the men part.

Reply:
Sure, done. I was afraid this would break things but at least loading a damaged CI from a MUL is compatible across this change.